app/assets/javascripts/models/beta/story.js

```diff
@@ -25,3 +25,7 @@ export const compareDeliveredAt = (a, b) => {
 export const compareStartedAt = (a, b) => {
   return compareValues(a.startedAt, b.startedAt);
 }
+
+export const filterUnestimatedFeatures = (story) => {
```

Comment:
I'd call this function `isUnestimatedFeature` because it doesn't really filter feature stories, it just returns if a story is a feature and is unestimated.